source/index.html

```diff
@@ -1,12 +1,157 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="page-header">
+      <div class="page-header__logo">
+        <a class="page-header__icon-logo">
+          <img class="page-header__image-icon logo-image" src="https://via.placeholder.com/150x150" width="33" height="38" alt="Supercat foto">
+        </a>
+        <a class="page-header__text-logo">
+          <img class="page-header__image-text logo-text" src="https://via.placeholder.com/150x150" width="101" height="18" alt="Логотип Cat energy">
+        </a>
+      </div>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
```

Comment:
```suggestion
          <span class="visually-hidden">Открыть меню.</span>
```
Надо ставить точки, тк скрин-ридер делает паузу на них. Это же законченное предложение.